com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.Streaming.cs

```diff
@@ -225,37 +213,56 @@ bool UpdateBlendingCellStreaming()
+            ComputeStreamingScoreForBlending(m_LoadedBlendingCells, true, worstScore);
 
             m_ToBeLoadedBlendingCells.QuickSort();
             m_LoadedBlendingCells.QuickSort();
 
-            int budget = m_BlendingPool.GetRemainingChunkCount();
-            int numberOfCellsToLoad = Mathf.Min(m_ToBeLoadedBlendingCells.size, m_NumberOfCellsLoadedPerFrame);
-            while (m_TempBlendingCellToLoadList.size < numberOfCellsToLoad)
+            int cellCountToLoad = Mathf.Min(m_NumberOfCellsLoadedPerFrame, m_ToBeLoadedBlendingCells.size);
+            while (m_TempBlendingCellToLoadList.size < cellCountToLoad)
             {
                 var blendingCell = m_ToBeLoadedBlendingCells[m_TempBlendingCellToLoadList.size];
-                if (!TryLoadBlendingCell(blendingCell, ref budget, m_TempBlendingCellToLoadList))
+                if (!TryLoadBlendingCell(blendingCell, m_TempBlendingCellToLoadList))
                     break;
             }
 
             // Budget reached
-            if (m_TempBlendingCellToLoadList.size != numberOfCellsToLoad)
+            if (m_TempBlendingCellToLoadList.size != cellCountToLoad)
             {
-                while (m_TempBlendingCellToLoadList.size < numberOfCellsToLoad)
+                // Turnover allows a percentage of the pool to be replaced by cells with a lower streaming score
+                // once the system is in a stable state. This ensures all cells get updated regularly.
+                int idx = (int)(m_LoadedBlendingCells.size * (1.0f - turnoverRate));
+                var worstNoTurnover = idx < m_LoadedBlendingCells.size ? m_LoadedBlendingCells[idx] : null;
+                while (m_TempBlendingCellToLoadList.size < cellCountToLoad)
                 {
                     if (m_LoadedBlendingCells.size - m_TempBlendingCellToUnloadList.size == 0) // We unloaded everything
                         break;
 
                     var worstCellLoaded = m_LoadedBlendingCells[m_LoadedBlendingCells.size - m_TempBlendingCellToUnloadList.size - 1];
                     var bestCellToBeLoaded = m_ToBeLoadedBlendingCells[m_TempBlendingCellToLoadList.size];
 
-                    if (bestCellToBeLoaded.streamingScore >= worstCellLoaded.streamingScore) // We are in a "stable" state
-                        break;
+                    if (bestCellToBeLoaded.streamingScore >= (worstNoTurnover ?? worstCellLoaded).streamingScore) // We are in a "stable" state
+                    {
+                        if (worstNoTurnover == null) // Disable turnover
+                            break;
 
-                    UnloadBlendingCell(worstCellLoaded, ref budget, m_TempBlendingCellToUnloadList);
-                    TryLoadBlendingCell(bestCellToBeLoaded, ref budget, m_TempBlendingCellToLoadList);
+                        int turnoverOffset = Random.Range(m_TempBlendingCellToLoadList.size, m_ToBeLoadedBlendingCells.size);
```

Comment:
cell is chosen randomly, not very satisifed with that but since elements from m_TempBlendingCellToLoadList are removed and added every frame, and the array is sorted, linear iterations skips elements at the start of the array

 Best option would be to sort the array to get the worst blended cells